cookbook/specs/spec_add_scroll.json

```diff
@@ -0,0 +1,118 @@
+{
+    "proposal": {
+        "title": "Add Specs: Scroll",
+        "description": "Adding new specification support for relaying Scroll zkEVM data on Lava",
+        "specs": [
+            {
+                "index": "SCRLL",
```

Comment:
"index": "SCROLL"
just one letter and it would make it more clear

Reply:
Sounds good, I'll make this change. I wasn't sure how strict the "remove vowels" rule was that was mentioned in the spec docs.